### PR TITLE
[infra] SANITIZER environment variable

### PR DESCRIPTION
**Pull request by mikea**
The variable picks one of the predefined flags configuration.

### CODE REVIEW

File: infra/base-images/base-libfuzzer/README.md [hunk 1]
```diff
@@ -32,6 +32,25 @@ docker run --rm -ti -v <b><i>$testcase_file</i></b>:/testcase ossfuzz/<b><i>$pro
...
+Build configuration is performed through following environment variables:
+
+| Env Variable     | Description
+| -------------    | --------
+| `SANITIZER` (`"address"`) | Specifies sanitizer configuration to use. `address` or `undefined`.
+| `SANITIZER_FLAGS` | Specify compiler sanitizer flags directly. Overrides `SANITIZER`.
+
+# Examples
+
+- *building sqlite3 fuzzer with undefined sanitizer:*
```
Comment (kcc) [inline infra/base-images/base-libfuzzer/README.md]: Maybe like this?
building sqlite3 fuzzer with UBSan (SANITIZER=undefined)

File: infra/base-images/base-libfuzzer/README.md [hunk 2]
```diff
@@ -32,6 +32,25 @@ docker run --rm -ti -v <b><i>$testcase_file</i></b>:/testcase ossfuzz/<b><i>$pro
...
+
+   <pre>
+docker run --rm -ti -e <i>SANITIZER</i>=<i>"undefined</i> ossfuzz/expat
```
Comment (kcc) [inline infra/base-images/base-libfuzzer/README.md]: did you mean sqlite or expat?

File: infra/base-images/base-libfuzzer/README.md (hunk 1 above)
Comment (mikea) [inline infra/base-images/base-libfuzzer/README.md]: done

File: infra/base-images/base-libfuzzer/README.md (hunk 2 above)
Comment (mikea) [inline infra/base-images/base-libfuzzer/README.md]: done

**Comment by kcc** [comment]
LGTM

**Pull request merged**